### ISSUE TITLE
FlowCrypt not available to use via TestFlight

### ISSUE THREAD

**Issue by seisvelas** [FlowCrypt/flowcrypt-ios#251]
From https://mail.google.com/mail/u/5/#inbox/FMfcgxwJXVGtMMMSjrxktTVhCWjdrgKp

> Hi,
> 
> I had iOS beta on my iPhone, but that phone went kaput and I had to switch to a new one. When I tried to install the iOS beta on the new phone (via TestFlight), I received a message saying No Apps Available to Test with instruction to contact developer for an invitation code. Is there such a code? Or is the beta now available now?

I'd try this myself but I don't have a device with iOS.

**Comment by codekittie** [comment]
Another user cannot find iOS beta with TestFlight:

>Right now, when I try to load FlowCrypt for iOS via Test Flight, there is a message that there is no beta version to test right now. Will there be one available again soon?

ref: https://mail.google.com/mail/u/human@flowcrypt.com/#inbox/FMfcgxwJZJbsHLrjZnbrdQxDMStqqXWQ

**Comment by seisvelas** [comment]
ref: https://mail.google.com/mail/u/human@flowcrypt.com/#inbox/FMfcgxwJZJXZZmtjBzmfltDbQhzrpSnC

**Comment by seisvelas** [comment]
>I was in the iOS beta but it appears to have expired. Is there a way for me to rejoin?

ref: https://mail.google.com/mail/u/human@flowcrypt.com/#inbox/FMfcgxwKjBNlQQQphrcwlzBvVlpKDRXz

**Comment by tomholub** [comment]
Yes, I'll need to do a new release soon - they expire in 90 days each time.

**Comment by seisvelas** [comment]
Please let me know here once the new release has been published so I can update these support emails letting users know :)

**Comment by seisvelas** [comment]
ref: https://mail.google.com/mail/u/human@flowcrypt.com/#inbox/FMfcgxwJZJbsHLrjZnbrdQxDMStqqXWQ

**Comment by seisvelas** [comment]
ref: https://mail.google.com/mail/u/human@flowcrypt.com/#inbox/FMfcgxwKjTPjhLWXKNsCWZmGgpRVTDnH

**Comment by tomholub** [comment]
thank you, yes I really need to fix this soon

**Comment by seisvelas** [comment]
ref: https://mail.google.com/mail/u/human@flowcrypt.com/#inbox/FMfcgxwKjTVSxgFdlxCgJNLWxVkmBvSH

>thank you, yes I really need to fix this soon

At least we can ascertain from the many inquiries that people are liking the beta :)

**Comment by seisvelas** [comment]
ref: https://mail.google.com/mail/u/human@flowcrypt.com/#inbox/FMfcgxwKjfBzVXHSKTsDKwfZnvxFFFsw

**Comment by seisvelas** [comment]
ref: https://mail.google.com/mail/u/human@flowcrypt.com/#inbox/FMfcgxwKjnbqdgvtQzMKglgwvXZkccQZ

Noteworthy: this is an enterprise user with Valeo

**Comment by seisvelas** [comment]
ref: https://mail.google.com/mail/u/human@flowcrypt.com/#inbox/FMfcgxwKjndBJKRLtRZjJMlJnXnhGnGb

**Comment by seisvelas** [comment]
ref: https://mail.google.com/mail/u/human@flowcrypt.com/#inbox/FMfcgxwKjnbrQFMTrNHgzbWSDwlXjlKq

**Comment by seisvelas** [comment]
ref: https://mail.google.com/mail/u/human@flowcrypt.com/#inbox/FMfcgxwKjngSMPgKnrzDDxCVkZTzwHZC

**Comment by seisvelas** [comment]
A user asked for ETA on when this will be available again, I wasn't sure what to tell them as I am not sure either. @tomholub any idea when this might be up again?

> Hi!
> 
> When do you expect the iOS version to be available? I was a beta tester at one point but lost my job due to the pandemic.
> 
> Looks as if the GitHub repo is showing a code update but TestFlight is saying it no longer accepting applicants?
> 
> Many thanks.

ref: https://mail.google.com/mail/u/human@flowcrypt.com/#inbox/FMfcgxwKkHhPWscjdvwmRVWDNMpVJrxG

**Comment by seisvelas** [comment]
ref: https://mail.google.com/mail/u/human@flowcrypt.com/#inbox/FMfcgxwKkRJSfXTGbsmttlHjlLsjbDnB

**Comment by seisvelas** [comment]
ref: https://mail.google.com/mail/u/human@flowcrypt.com/#inbox/FMfcgxwLsJvLLXlvKvXRrjLRCjFvsNGW

**Comment by tomholub** [comment]
Thank you, I really need to do a release soon

**Comment by seisvelas** [comment]
ref: https://mail.google.com/mail/u/human@flowcrypt.com/#inbox/FMfcgzGkXSTdrbkhSntfsXGcmLpGsPDx

**Comment by seisvelas** [comment]
>There were 1 email addresses found in the above comment. Please:

Ha - with the new exemption feature from https://github.com/FlowCrypt/flowcrypt-android/pull/1244, we can add an exemption for flowcrypt.com so support issues don't trigger the monitoring

**Comment by seisvelas** [comment]
ref: https://mail.google.com/mail/u/human@flowcrypt.com/#inbox/FMfcgzGkXwFWFvHRnNMSpMhpdpBRlDnJ

**Comment by seisvelas** [comment]
https://mail.google.com/mail/u/human@flowcrypt.com/#inbox/FMfcgzGkZGqmtTChVzRMMwCrjsfPpmxH

>(uninstall feedback) need it to also work on iphone

**Comment by seisvelas** [comment]
https://mail.google.com/mail/u/human@flowcrypt.com/#inbox/FMfcgzGkZQGRwvrWkTRzSqhQdRGSdrxF

>Please send me the IOS beta when released

**Comment by seisvelas** [comment]
https://mail.google.com/mail/u/human@flowcrypt.com/#inbox/FMfcgzGkZQKtMtdCswvWsfZVBFflqmJL

Another user who would like me to alert them when the beta is back (I'm putting them all here so I remember to email them)

@tomholub I'm curious, is there a specific feature or something similar we're waiting for before the beta is ready to be released?

**Comment by tomholub** [comment]
Not really, it's waiting for my availability to do a release, and to resolve some administrative issues on our apple account to do so (it's registered for a wrong company entity which should be fixed soon).
